src/kamiye/feature.js

```diff
@@ -1 +1,85 @@
-console.log('Oluwakamiye Adelemoni');
\ No newline at end of file
+console.log('Oluwakamiye Adelemoni');
+
+/*
+Activity One (data types)
+=========================
+What data type:
+
+1. 14 = number
+2. -80 = number
+3. 8.5 = number
+4. true = boolean
+5. (5<17) = boolean
+6. null = undefined
+7. 3.142 = number
+8. 'pi' = string
+9. 67+18 = number
+*/
+
+
+
+
+
+
+
+//Question: write a program to produce binary values from decimal.
+
+/*
+Solution gotten from Josh's algorithm
+*/
+
+var binaryConverter = function(number){
+	var reverseBinary = [];
+	do{
+		reverseBinary.push(number%2);
+		number = Math.floor(number/2);
+		console.log(number);
+	}while(number>=1)
```

Comment:
For more readable code, it's recommended to surround operators with spaces. Also, there should be a space between a keyword and a brace.